src/artemis/devices/unit_tests/test_gridscan.py

```diff
@@ -0,0 +1,134 @@
+from ophyd.sim import make_fake_device
+from src.artemis.devices.fast_grid_scan import FastGridScan, time
+
+from mockito import *
```

Comment:
Personally I prefer to import what you use rather than `from thing import *` - so for example when I wonder what `when()` is below, I can see where it comes from

Reply:
Agreed, I wouldn't use `import *` in the actual production code so I shouldn't in the test code.